lens/carrepo/carrepo.go

```diff
@@ -35,3 +38,56 @@ func NewAPIOpener(c *cli.Context) (lens.APIOpener, lens.APICloser, error) {
 
 	return util.NewAPIOpener(c.Context, cacheDB, h, c.Int("lens-cache-hint"))
 }
+
+type wrapper struct {
+	c *carbs.Carbs
+}
```

Comment:
Wrapper required since carbs [doesn't implement](https://github.com/willscott/carbs/blob/master/carbs.go#L25) the [extended blockstore interface](https://github.com/filecoin-project/lotus/blob/4c05ec28ba45e6947118c5a0bc836f312442d0e9/blockstore/blockstore.go#L20-L21) expected by lotus.

Lotus interface requires `View` and `DeleteMany` methods.